src/Combinatorics/EnumerativeCombinatorics/compositions.jl

```diff
@@ -152,11 +152,11 @@ parts(C::CompositionsFixedNumParts) = C.k
 Base.eltype(C::CompositionsFixedNumParts{T}) where T = Composition{T}
 
 function Base.show(io::IO, C::CompositionsFixedNumParts)
-  if get(io, :supercompact, false)
-    print(io, "Iterator")
+  if is_terse(io)
+    print(io, "Iterator over the compositions of $(base(C)) into ", ItemQuantity(parts(C), "parts"))
   else
     io = pretty(io)
-    print(io, "Iterator over the compositions of $(base(C)) into ", ItemQuantity(parts(C), "part"))
+    print(io, "Iterator over the compositions of $(base(C)) into ", ItemQuantity(parts(C), "parts"))
   end
```

Comment:
This should just be
```
print(pretty(io), "Iterator over the compositions of $(base(C)) into ", ItemQuantity(parts(C), "part"))
```
without the `if`/`else`. And the `ItemQuantity` takes care of the plural s, so `part` is correct here :)

Same below.

Reply:
ah okay, my mistake. fixed now

Reply:
Thank you!